src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Test-Json command.
+    /// </summary>
+    [Cmdlet(VerbsDiagnostic.Test, "Json", HelpUri = "")]
+    public class TestJsonCommand : PSCmdlet
+    {
+        /// <summary>
+        /// An Json to be validated.
+        /// </summary>
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true)]
+        public String Json { get; set; }
+
+        /// <summary>
+        /// A schema to validate the Json against.
+        /// </summary>
+        [Parameter(Position = 1)]
+        [ValidateNotNullOrEmpty()]
+        public String Schema { get; set; }
+
+        private JSchema _jschema = null;
```

Comment:
[Nitpick] We normally don't explicitly initialize fields to null since C# guarantees fields are initialized to zero.

Reply:
@daxian-dbw It seems we should add this in our code guidance.

Reply:
The initializer was removed.